_posts/2022-08-29-hello-evvnt.markdown

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title:  "Hello Evvnt"
+date:   2022-08-29 15:50:14 -0400
+categories: engineering
+---
+Welcome to the Evvnt Engineering Blog! This is our hello world introducing Evvnt and what we do. First thing is first, its not a typo! The company name is Evvnt and its pronounced Event. (We don't blame you if you pronounce it like [marwage](https://www.youtube.com/watch?v=3odMTPuzLwY).)
+
+Our Passion: **Powering Experiences to Connect People**
+
+*People Value Social Interaction Through Local Events that Vitalize Communities and Improve Life Quality*
+
+Q: Why haven't I heard of Evvnt?
+A: Today we deliver our software as white labeled. We are on thousands of media sites power event calendars and selling millions of dollars in tickets.
+
+Q: What makes Evvnt different than any other software company out there?
+A: We believe in the following values that we embody every day:
+* *Transparency* - we talk about what we do and why at every level in the company
+* *Respect* - we believe that each and every person deserves respect and we demonstrate it daily.
+* *Ownership* - we believe that a team that owns the company and outcomes is more motivated and makes better decisions.
+* *Quality at Pace* - we believe that quality wins over speed and when combined in what we call "Always be shipping". We can build great things that are sustainable and compounding.
```

Comment:
*Quality at Pace* - we believe that quality wins over speed and our development philosophy is "To Always Be Shipping" (ABS). We goal is to build great things that are sustainable and compounding.